README.md

```diff
@@ -20,6 +20,7 @@ special thanks to [@timgrossmann](https://github.com/timgrossmann) for creating
 
 - install [nodejs](https://nodejs.org)
 - open the terminal and create a new folder
+	- the name the folder different than 'jinsta'
```

Comment:
_the name of the folder must be different than 'jinsta'_

Reply:
ouh :O thanks alot :D